R/importCNA.GISTIC2.R

```diff
@@ -0,0 +1,105 @@
+# importCNA.GISTIC2.R
+#
+#
+# Imports copy number aberration data from Firehose database,
+# produces an rCNA object saved as a RDS file.
+#
+#
+#'@param fFHG A vector of local fully qualified paths of Firehose GISTIC2 CNA file(s).
+#'@param dCNA The local fully qualified path of a directory to store the rCNA RDS file(s).
+#'@param silent Boolean. Default: FALSE. Whether or not to write progress information to console.
```

Comment:
I think the function should operate on one file at a time. The user should supply a valid filename, not merely a directory. Otherwise you would need a large mechanism for creating unique filenames - and that's really not the job of this function.